app/models/subnet.rb

```diff
@@ -216,6 +220,12 @@ def unused_ip(mac = nil, excluded_ips = [])
       end
       logger.debug("Not suggesting IP Address for #{self} as no free IP found in our DB")
       return
+    elsif self.ipam == IPAM_MODES[:eui64] && self.class.supports_ipam_mode?(:eui64)
```

Comment:
db will also be in ipv6, it had to remove it because it needs changes to the host or nic model...
I think, I could move that to a dedicated class or something. But the question is: Where to start and when to stop... I'll take a look and see if i can improve that without much effort...